recipes/safe-pysha3/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "safe-pysha3" %}
+{% set repo_name = "pysha3" %}
+{% set version = "1.0.4" %}
+{% set sha256 = "0398e0b98573d93c81318073c2d2c2426a6d02b4bca85ef483c4276bdda3378c" %}
```

Comment:
```suggestion
```

Reply:
Right, this recipe slipped through the refactoring from LL of previous ones